CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+## Version 3.4.18 GA (10/30/2023)
+
+### Enhancements:
+
+* Update to OpenTelemetry Java 1.31 ([#3341](https://github.com/microsoft/ApplicationInsights-Java/pull/3341))
+* Update to azure-monitor-opentelemetry-exporter to 1.0.0-beta.13 ([#3356](https://github.com/microsoft/ApplicationInsights-Java/pull/3356))
+* Add support for otel.metrics.exporter=otlp ([#3301](https://github.com/microsoft/ApplicationInsights-Java/pull/3301))
```

Comment:
let's consider this internal only feature until we are ready to document it publicly
```suggestion
```